README.md

```diff
@@ -1,137 +1,104 @@
-| rel/1.0.1          | 1.0.1            | [![Build Status](https://travis-ci.org/aspnet/JitBench.svg?branch=rel/1.0.1)](https://travis-ci.org/aspnet/JitBench)          |
-
-## Instructions for JIT testing:
-
-### Step 0:
-
-Clone the JitBench Repo
-
-`git clone <JitBench repo>`
-
-`cd JitBench`
-
-### Step 1:
-
-Get the newest dotnet Shared Runtime as 'repo local' 
-
-**Windows**
-
-`.\Dotnet-Install.ps1 -SharedRuntime -InstallDir .dotnet -Channel master -Architecture x64`
-
-`.\Dotnet-Install.ps1 -InstallDir .dotnet -Channel master -Architecture x64`
-
-**OSX**
-
-`./dotnet-install.sh -sharedruntime -runtimeid osx.10.12-x64 -installdir .dotnet -channel master -architecture x64` 
-
-`source ./dotnet-install.sh -installdir .dotnet -channel master -architecture x64`
+## Example benchmark usage with powershell
 
-**Linux**
+### First get the repo
 
-`./dotnet-install.sh -sharedruntime -runtimeid linux-x64 -installdir .dotnet -channel master -architecture x64` 
+    git clone <JitBench repo>
+    cd JitBench
 
-`source ./dotnet-install.sh -installdir .dotnet -channel master -architecture x64`
+### Example 1: Test using 2.0 RTM versions of all .Net Core components
 
+    .\RunBenchmark.ps1
 
-You need to run **both** of these commands in this particular order. This will grab the latest shared runtime and SDK and copy them to `<JitBench>\.dotnet`. Note you need to `source` the second script so that it can update your `$PATH`.
+Outputs:
 
-You should also have this version of `dotnet` on your path at this point. `dotnet --info` will print the version and it should match what you see in the output of the above commands.
+	... lots of logging omitted, then at the bottom MusicStore runs ...
 
-### Step 2:
+	Server started in 2847ms
+	
+	Starting request to http://localhost:5000
+	Response: OK
+	Request took 618ms
+	
+	Cold start time (server start + first request time): 3465ms
+	
+	
+	Running 100 requests
+	Steadystate min response time: 3ms
+	Steadystate max response time: 8ms
+	Steadystate average response time: 3ms
+	
+	ASP.NET loaded from store
 
-Modify the shared framework (if necessary).
+### Example 2: Test with a custom version of the shared framework
 
-If you need to use a private build of the JIT or other CoreCLR components, now is a good time to update the shared framework with your bits. Copy any binaries you need to use into the shared framework in `<JitBench>\.dotnet\shared\Microsoft.NETCore.App\<version>`. The version should match the version that downloaded in step 1.
+    .\RunBenchmark.ps1 -FrameworkVersion 2.1.0-preview1-25818-02
 
-### Step 3a:
+This will automatically select an LKG version of the SDK capable of working with this version of the framework because 2.0.0 SDK won't work. In this case, SDK 2.2.0-preview1-007558 is selected and printed in the log. ASP.Net remains 2.0.0.
```

Comment:
So what tracks updating this LKG - https://github.com/aspnet/JitBench/pull/83/files#diff-ce2f9e0f399da6bb1f4aaaf11796d06cR33

If you (or someone else working on the CLR) is going to be fairly proactive in managing the version that's fine. We could also plan to just update the SDK each time it releases.

Reply:
My expectation is that someone on my team will update it as needed if we need a new version of some component and it isn't compatible the LKG SDK recorded there. My hope is that this will be a rare occurrence (once or twice per release cycle?)